src/minter-contract-utils/src/bft_bridge_api.rs

```diff
@@ -249,9 +249,57 @@ pub enum BridgeEvent {
 impl BridgeEvent {
     pub async fn collect_logs(
         evm_client: &EthJsonRpcClient<impl Client>,
+        mut from_block: u64,
+        blocks_to_fetch: u64,
```

Comment:
this should be renamed `to_blocks` and should be Optional (if 'None`, use `Block::Latest`)